### PR TITLE
@feature/video player

### PR DESCRIPTION
**Pull request by ukirderohit**
Video player with seeker added

### CODE REVIEW

File: packages/im-base/src/utils/getFormattedTime.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const getFormattedTime = (seconds) => {
+  const date = new Date(seconds * 1000);
+  const hh = date.getUTCHours();
+  const mm = date.getUTCMinutes();
+  const ss = pad(date.getUTCSeconds());
```
Comment (pranay2401) [inline packages/im-base/src/utils/getFormattedTime.js]: should be a more meaningful name

File: packages/im-components/src/organisms/seeker/Seeker.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  const [duration, setDuration] = useState(true);
+
+  const [played, setPlayed] = useState(0);
+
+  // seeking video by input range
+  const seekVideo = (e) => {
+    if (!playing) {
+      setPlaying(true);
+    }
+    if (e.target?.value) {
```
Comment (pranay2401) [inline packages/im-components/src/organisms/seeker/Seeker.js:30]: should be more meaningful name for readability - e

File: packages/im-base/src/utils/getFormattedTime.js (hunk 1 above)
Comment (ukirderohit) [reply packages/im-base/src/utils/getFormattedTime.js]: which one hours, mins, seconds ? or padding "0"?

File: packages/im-components/src/organisms/seeker/Seeker.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+};
+
+Seeker.propTypes = {
+  className: PropTypes.string,
+  videoUrl: PropTypes.string,
+};
+
+Seeker.defaultProps = {
+  className: undefined,
+  videoUrl: "https://www.youtube.com/watch?v=vUjihJpWQPE",
```
Comment (pranay2401) [inline packages/im-components/src/organisms/seeker/Seeker.js:83]: why this default video link?

File: packages/im-ott/src/routes/AppRoutes.js [hunk 4]
```diff
@@ -29,6 +31,8 @@ const AppRoutes = () => {
         <Route exact path="/" component={LandingPage} />
         <Route exact path="/watchlist" component={WatchList} />
         <Route exact path="/film/:filmId" component={FilmDetails} />
+        <Route exact path="/player" component={Player} />
```
Comment (pranay2401) [inline packages/im-ott/src/routes/AppRoutes.js:34]: Do we need a separate path for player and seeker ?

File: packages/im-components/src/organisms/seeker/Seeker.js (hunk 3 above)
Comment (ukirderohit) [reply packages/im-components/src/organisms/seeker/Seeker.js:83]: it's just for time being to test... once we reuse we will provide that video URL

**Review by pranay2401 — CHANGES_REQUESTED** [review]
Please add changes

File: packages/im-ott/src/routes/AppRoutes.js (hunk 4 above)
Comment (ukirderohit) [reply packages/im-ott/src/routes/AppRoutes.js:34]: to test, should i remove all test links?

File: packages/im-base/src/utils/getFormattedTime.js (hunk 1 above)
Comment (pranay2401) [reply packages/im-base/src/utils/getFormattedTime.js]: pad

File: packages/im-ott/src/routes/AppRoutes.js (hunk 4 above)
Comment (pranay2401) [reply packages/im-ott/src/routes/AppRoutes.js:34]: It's okay to keep it for now. Just wanted to know the reason

File: packages/im-components/src/organisms/overlay/overlay.module.scss [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+      button:first-child {
+        position: relative;
+        margin-top: 2rem;
+        margin-right: 2.5rem;
+      }
+    }
+
+    .selected {
+      border: 2px solid green;
```
Comment (pranay2401) [inline packages/im-components/src/organisms/overlay/overlay.module.scss]: convert to rem

File: packages/im-components/src/organisms/player/Player.js [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+// Components
+import screenfull from "screenfull";
+import ReactPlayer from "react-player/youtube";
+import Overlay from "../overlay";
+
+// Styles
+import styles from "./player.module.scss";
+
+// Mock Data
+const overlayData = [
```
Comment (pranay2401) [inline packages/im-components/src/organisms/player/Player.js]: This can  be added to separate file and used directly for test cases.

**Review by pranay2401 — CHANGES_REQUESTED** [review]
Please add changes.

**Comment by ukirderohit** [comment]
> Please add changes.

Done

**Review by pranay2401 — APPROVED** [review]

**Pull request merged**